src/oph/ehoks/palaute/scheduler.clj

```diff
@@ -46,10 +42,23 @@
       (Instant/from)))
 
 (def schedules
-  [{:action daily-actions!
-    :start (time->instant 6 0 0) :period (Period/ofDays 1)}
-   {:action hourly-actions!
-    :start (time->instant 1 0 0) :period (Duration/ofHours 1)}])
+  [{:start (time->instant 6 0 0) :period (Period/ofDays 1)
+    :action daily-actions!}
+
+   {:start (time->instant 1 0 0) :period (Duration/ofHours 1)
+    :action hourly-actions!}
+
+   {:start (time->instant 1 0 0) :period (Duration/ofMinutes 5)
```

Comment:
Tässä tää time->instant-systeemi ei oo ehkä enää ihan se mitä halutaan, koska se siis jonouttaa kaikki viisiminuuttiset mitä on ehtinyt kertyä klo 01:00 samana päivänä jälkeen.  Tää tapahtuu vain palvelua käynnistäessä, sitten kun jono on saatu tyhjäksi niin se tapahtuu kyllä nätisti 5 minuutin välein siitä eteenpäin.

Täl ei mun mielestä oo niin paljon väliä että tälle tarttis tehdä jotain.